src/backend/base/pyproject.toml

```diff
@@ -194,6 +195,8 @@ select = [
 ignore = [
     "COM812", # Messes with the formatter
     "ISC001", # Messes with the formatter
+    "RUF006", # TODO (Store a reference to the return value of `asyncio.create_task`)
+    "RUF012", # TODO (Mutable class attributes should be annotated with `typing.ClassVar`)
```

Comment:
At the moment I ignore these rules as they deserve some discussion.

Reply:
As for RUF012, there seems to be an issue in ruff not detecting pydantic models. Ruff should normally ignore pydantic models for this rule as pydantic does a deep copy of default objects.

Reply:
I opened https://github.com/astral-sh/ruff/issues/13630